src/arrays/programs_involving_data_sequences/easy/q6/MainTest.java

```diff
@@ -0,0 +1,116 @@
+package arrays.programs_involving_data_sequences.easy.q6;
+
+import global.BaseTest;
+import global.MethodTest;
+import global.tools.CustomAssertions;
+import global.tools.TestOption;
+import global.utils.ArrayUtil;
+import global.variables.Clause;
+import global.variables.clauses.DoubleLiteral;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MainTest extends BaseTest {
+    // Parsons
+
+    //vary this variable to vary number of bmis calculated. Max of 20 (inclusive) because of default input.
+    final static int size = 10;
+
+    public Clause[] testSentence() {
+        TestOption.isInputTest = true;
+        TestOption.defaultInput = "180 65 135 45 187.3 95 67 23 132 78.3 184.78 67.521 704 15.4 63.5 38 374 23.4 437 38 180 65 135 45 187.3 95 67 23 132 78.3 184.78 67.521 704 15.4 63.5 38 374 23.4 437 38";
```

Comment:
We should be able to just generate this default input with arrayUtil and then we don't need this limit on the variation,

Reply:
As it doesn't really matter what the default is